db/migrate/20250215153930_add_exchange_operating_mic_col_label_to_imports.rb

```diff
@@ -0,0 +1,16 @@
+class AddExchangeOperatingMicColLabelToImports < ActiveRecord::Migration[7.2]
+  def up
+    add_column :imports, :exchange_operating_mic_col_label, :string
+
+    # Migrate existing trade imports to use the new column
+    Import.where(type: "TradeImport").find_each do |import|
+      if import.exchange_col_label.present?
+        import.update_column(:exchange_operating_mic_col_label, import.exchange_col_label)
+      end
+    end
```

Comment:
I think we can skip this step of migrating old imports and just let the user fix it manually.  Most users will likely need to reconfigure their import data anyways.